src/rc/seedrng.c

```diff
@@ -392,10 +441,30 @@ int main(int argc _unused, char *argv[] _unused)
 	struct timespec realtime = { 0 }, boottime = { 0 };
 	struct blake2s_state hash;
 
+	atexit(cleanup);
+	applet = basename_c(argv[0]);
+	if (geteuid())
+		eerrorx("%s: superuser access is required", applet);
```

Comment:
The `"%s: ...` prefix looks nice and seems reasonable. But I noticed that the other eerror and ewarn statements don't have the `%s: ` prefix in there. I don't know if some types of messages need this and others don't, but maybe this is inconsistent. Up to you, so just FYI.